lib/index.ts

```diff
@@ -128,7 +144,7 @@ async function getDependencies(root, targetFile) {
   let tempDirObj;
   const packageManager = pkgManagerByTarget(targetFile);
   if (packageManager === 'gomodules') {
-    return buildDepTreeFromImportsAndModules(root, targetFile);
+    return { dependencyGraph: await buildDepGraphFromImportsAndModules(root, targetFile)};
```

Comment:
add return type to function definition?

Reply:
Sure, good catch Promise DepTree | DepGraph, thanks! :)